vue-components/stories/BouncingDots.stories.ts

```diff
@@ -0,0 +1,39 @@
+import BouncingDots from '@/components/BouncingDots';
+import { bouncingDotsSizes } from '@/components/bouncingDots';
+import { Component } from 'vue';
+
+export default {
+    component: BouncingDots,
+    title: '/Loader/BouncingDots',
+};
+
+export function basic( args: { content: string; underlined: boolean } ): Component {
+    return {
+        data(): object {
+            return { args, bouncingDotsSizes };
+        },
+        components: { BouncingDots },
+        props: Object.keys( args ),
+        template: `
+			<div>
+				<div v-for="size in bouncingDotsSizes">
```

Comment:
```suggestion
				<div v-for="size in bouncingDotsSizes" style="font-family: sans-serif; color: #202122">
```

Reply:
I'm hoping this is the way to modify the text style of the size indicator. 

I was trying to do this tinny change locally, but I'm getting the following error (probably irrelevant):

![Screenshot 2021-03-11 at 10 17 47](https://user-images.githubusercontent.com/3887458/110764182-086a1b00-8253-11eb-96ca-7ea132824503.png)